data/rao-result/rao-result-json/src/main/java/com/powsybl/openrao/data/raoresultjson/RaoResultJsonConstants.java

```diff
@@ -99,6 +99,8 @@ private RaoResultJsonConstants() {
     // branch side
     public static final String LEFT_SIDE = "leftSide";
     public static final String RIGHT_SIDE = "rightSide";
+    public static final String SIDE_ONE = "sideOne";
+    public static final String SIDE_TWO = "sideTwo";
```

Comment:
let's make RAO result lighter
```suggestion
    public static final String SIDE_ONE = "side1";
    public static final String SIDE_TWO = "side2";
```

Reply:
done